### ISSUE TITLE
Add a UUID Blacklist function

### ISSUE THREAD

**Issue by sedorr** [CarlosEsco/Neko#1035]
### Describe your suggested feature

MangaDex allows you to Blacklist users or Scanlation groups by UUID, Tachyomi does so as well. 

I normally always blacklist BiliBili comics as they tend to upload already translated chapters for comics they sell thus prompting the app to give you an update notification for old chapters you`ve already read ages ago.

I know that you can filter by scanlators but that doesnt affect updates or the unread bubble.

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open issue.
- [X] I have written a short but informative title.
- [X] I have updated the app to the newest version **[Latest](https://github.com/CarlosEsco/Neko/releases/latest)**.
- [X] I have checked through the app settings for my feature.
- [X] I will fill out all of the requested information in this form.

**Comment by CarlosEsco** [comment]
I do plan to add this eventually, I just might make it a client side filter so its easier for users to block groups.  There is no eta on this though

**Comment by goldbattle** [comment]
See this PR for ref: https://github.com/tachiyomiorg/tachiyomi-extensions/pull/10376

**Comment by CarlosEsco** [comment]
2.10.12

Neko does it differently.  Long press a chapter click block scanlator then click the group.  
If you want to unblock go into the mangadex site settings
